viewer/packages/cad-model/src/wrappers/CogniteCadModel.test.ts

```diff
@@ -114,4 +114,19 @@ describe(CogniteCadModel.name, () => {
 
     expect(model.getCdfToDefaultModelTransformation()).toEqual(originalSourceTransform);
   });
+
+  test('visible property hides or unhides model', () => {
+    const visible = false;
```

Comment:
For readability reasons I would switch this to true. That way "not to be visible" and "to be visible" can be read straight forward and it would make sense as it is.

```
const visible = true;
expect(model.visible).toBeTrue();

model.visible = false;

expect(model.cadNode.visible).not.toBe(visible);
expect(model.visible).not.toBe(visible);

model.visible = true;

expect(model.cadNode.visible).toBe(visible);
expect(model.visible).toBe(visible);
```